kasa/smartcam/modules/camera.py

```diff
@@ -29,28 +31,37 @@ class StreamResolution(StrEnum):
 class Camera(SmartCamModule):
     """Implementation of device module."""
 
-    QUERY_GETTER_NAME = "getLensMaskConfig"
-    QUERY_MODULE_NAME = "lens_mask"
-    QUERY_SECTION_NAMES = "lens_mask_info"
-
     def _initialize_features(self) -> None:
         """Initialize features after the initial update."""
-        self._add_feature(
-            Feature(
-                self._device,
-                id="state",
-                name="State",
-                attribute_getter="is_on",
-                attribute_setter="set_state",
-                type=Feature.Type.Switch,
-                category=Feature.Category.Primary,
+        if Module.LensMask in self._device.modules:
+            self._add_feature(
```

Comment:
Should this be initialized inside the module itself?

Reply:
As this is the `state` feature for the whole camera module I figure there could be other things we may want to use to describe camera on/off state. For example switching off motion detection on a battery camera that is not constantly recording could also be used as the `camera` `state`.

Reply:
Okay, let's see about that when we get that far, ready to be merged!